src/blockchain/ain/jellyfish/utils/raw-tx-utxo.ts

```diff
@@ -0,0 +1,96 @@
+import { Vout, Script } from '@defichain/jellyfish-transaction';
+import BigNumber from 'bignumber.js';
+import { Config } from 'src/config/config';
+import { UtxoConfig } from '../domain/entities/utxo-config';
+import { UtxoSizePriority } from '../domain/enums';
+import { RawTxDto } from '../dto/raw-tx.dto';
+import { RawTxBase } from './raw-tx-base';
+import { RawTxUtil } from './raw-tx-util';
+
+export class RawTxUtxo extends RawTxBase {
+  // SEND UTXOS //
+  async sendFromTo(from: string, to: string, amount: BigNumber): Promise<RawTxDto> {
+    return this.handle(() =>
+      this.send(from, to, amount, true, { useFeeBuffer: false, sizePriority: UtxoSizePriority.SMALL }),
+    );
+  }
+
+  async sendFromLiq(to: string, amount: BigNumber, sizePriority: UtxoSizePriority): Promise<RawTxDto> {
+    return this.handle(() =>
+      this.send(Config.staking.liquidity.address, to, amount, true, { useFeeBuffer: true, sizePriority }),
+    );
+  }
+
+  async sendToLiq(from: string, amount: BigNumber): Promise<RawTxDto> {
+    return this.handle(() =>
+      this.send(from, Config.staking.liquidity.address, amount, false, {
+        useFeeBuffer: false,
+        sizePriority: UtxoSizePriority.SMALL,
+      }),
+    );
+  }
+
+  private async send(
+    from: string,
+    to: string,
+    amount: BigNumber,
+    useChangeOutput: boolean,
```

Comment:
`useChangeOutput` might be better named `!useExactUtxo`?

Reply:
it depends how you read it. this boolean changes two outcomes
1. use of another provider function
2. number of output

For me it is more important to know, if it should have a change output or not. At least for me this change don't make it more readable. Not meaning that the current state is perfect